src/Header/Header.css

```diff
@@ -0,0 +1,33 @@
+.header {
+  background: linear-gradient(90deg, #672280 1.18%, #A626D3 100%);
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+
+}
+
+.header-left {
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+}
+
+.header-logo {
+  width: 50px;
+  height: 45px;
+  align-self: center;
+}
+
+.header-left-text {
+  font-family: 'Karla', sans-serif;
+  font-weight: 700;
+  font-size: 25px;
+  line-height: 24px;
```

Comment:
- [ ]  -  `line-height` has no use here. please remove it.

Reply:
Resolved